packages/rest/package-lock.json

```diff
@@ -0,0 +1,796 @@
+{
```

Comment:
AFAIK, we are not using package-lock files in loopback-next, see https://github.com/strongloop/loopback-next/blob/9f3a6f89958fd619aa3b1f5a002666e603b683fe/.npmrc#L1

Please remove this file.